### DISCUSSION TITLE
How to add VAE manually?

### DISCUSSION THREAD

**Discussion by CarbonMatrix** [TheLastBen/fast-stable-diffusion#1585] [Q&A] [answered]
I tried to use other VAEs.
My VAE is in the folder "/content/drive/MyDrive/stable-diffusion-webui/models/VAE", I run the fast_stable_diffusion_AUTOMATIC1111.ipynb from the folder "/content/drive/MyDrive/stable-diffusion-webui". 
Still, if I go to Settings > Stable Diffusion and reload "SD VAE", it still just shows "Automatic" and "None". I can't find my own VAEs. 
The file names of the VAE are correct.
Any help?
Thanks!

EDIT: 
I realized there is a "sd" folder created in /gdrive (not drive) as soon as you install the Automatic1111 repo (which is ofc temporary). If you copy the VAE in the "VAE" folder there (/content/gdrive/MyDrive/sd/stable-diffusion-webui/models/VAE), it works! But you have to upload the VAE every single time you start a session. Isn't there a better method to use a path to your permanent drive like with the model so you don't have to upload it every single time again?

**Comment by CarbonMatrix** [comment, marked as answer]
I answered it myself. I changed the first part to:
```
from google.colab import drive
drive.mount('/content/gdrive/', force_remount=True)
```
which forces it to use my gdrive and not create a different temporary gdrive folder.

**Comment by eliasprompt** [comment]
The first time you run the program it doesn't detect the VAEs in the VAE folder, but if you restart then it does. This didn't happen before and it's a bit inconvenient to have to run the program twice every time you install it, it's been like this for several weeks and that didn't happen before, I hope they fix it
